src/qt/governancelist.cpp

```diff
@@ -396,3 +475,25 @@ void GovernanceList::showAdditionalInfo(const QModelIndex& index)
 
     QMessageBox::information(this, windowTitle, json);
 }
+
+void GovernanceList::openUrl()
+{
+    QAction *openProposalUrl = qobject_cast<QAction*>(sender());
```

Comment:
ptr should go on the type
```suggestion
    QAction* openProposalUrl = qobject_cast<QAction*>(sender());
```